src/main/webapp/js/components/widgets/NewWidget.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * New Widget resizable component
+ * 
+ *  @author Dario Del Piano
+ */
+
+define(function (require) {
+
+    var React = require('react');
+    var Rnd = require('react-rnd').default;
+    require('./Widget.less');
+
+    class NewWidget extends React.Component {
```

Comment:
The concept is great, does it work? are we using it? cc @adrianq a widget that doesn't use JQuery UI for the dialog

Reply:
Screenshot? :)

Reply:
@tarelli removed from this branch as discussed.
@adrianq that was just a skeleton to start it, Rnd is the same library I used for the TabbedDrawer, in principle should work fine and most of the code done for the drawer can be re-used (maximize, close, minimize), drag and resizing work by default if enabled in the props we pass to Rnd.
The parts to implement will be the navigation bar with history and the arrows, when I will have a bit of time to invest in #INFRA I will provide a screenshot :)

Reply:
I see... that is the main problem we have to get rid of the current widget. It is not anymore just a floating dialog but it provides a lot of useful features that are actually fully tested. Looking forward to that moment in time we remove jquery-ui from Geppetto ;)